docs/example/saxml/README.md

```diff
@@ -0,0 +1,100 @@
+# Deploy SaxML Multihost with LWS on GKE
+
+In this example, we will use LeaderWorkerSet to deploy a multi-host inference instance on TPUs with Saxml. You could use the steps [here](https://cloud.google.com/kubernetes-engine/docs/tutorials/tpu-multihost-saxml#before-you-begin) to setup your TPU clusters, node pools, GCS bucket, and configure workload access. 
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+## Conceptual View
+
+![image](https://github.com/kubernetes-sigs/lws/assets/86417275/2c4a9abd-f988-4f2e-ad3f-c7b03f0b6485)
+
```

Comment:
we need to say somewhere that they have to replace `GCS_BUCKET` with the actual bucket value name in the yamls

Reply:
Added